libsync/SyncMaster.h

```diff
@@ -312,6 +323,11 @@ class SyncMaster : public SyncInterface, public Worker
     // verify handler to check downloading block
     std::function<bool(dev::eth::Block const&)> fp_isConsensusOk = nullptr;
 
+    dev::flowlimit::QPSLimiter::Ptr m_bandwidthLimiter;
+    dev::flowlimit::QPSLimiter::Ptr m_channelBandwidthLimiter;

```

Comment:
why sync has `m_channelBandwidthLimiter`? In my opinion channel has no releation with libsync

Reply:
Has already updated to m_nodeBandwidthLimiter.